Monika After Story/game/script-topics.rpy

```diff
@@ -2777,6 +2777,21 @@ label monika_eternity:
     m "You saved my life, and I'll forever be thankful."
     return
 
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['metal gear' , 'wolbachia' , 'kojima' , 'konami']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_mgs') #id
+    monika_random_topics.append('monika_sleep')
+
+label monika_sleep:
```

Comment:
Topic is fine. Might want to change the reference to something that wouldn't involve the game canon-wise i.e. scrapped designs, possible hidden stuff.